### DISCUSSION TITLE
Plotting energy bill

### DISCUSSION THREAD

**Discussion by pluto2021** [Chia-Network/chia-blockchain#2848] [General]
As plotting is becoming the real bottle neck, in the foreseeable future, we can get an estimated energy bill to plot out the entire netspace, both existing, and extrapolated future netspace.

Assuming that a typical plotting rig would get 30 plots a day with 500W power consumption, it would lead to 0.4 kWh per plot, or 4Mega Wh per PiB.  Now the netspace is about 850 PiB, that would have consumed about 3.4 GWh of electricity so far. This is about 1/3 of daily electricity New York city is using. In about 2 weeks the electricity consumption would equal 3 days of electricity consumption in New York city. In about a month from now, the plotting activity would have consumed one month worth of electricity consumption in New York City at 73EiB. If the growth stays the same, as best we can tell now, in just about only 2.5months from today, there could be enough Chia HDD space to match the total data size on the entire internet today! and the electricity consumed could skyrocket to about 60 New York Cities' electricity consumption in one entire year, but now squeezed into 2.5months! or 288 New York Cities in a same time frame. There are less number of cities on earth with that kind of size! 

As such, the current space growth rate might not be sustainable. This would really be testing where the practical limit of HDD space that are farmable.

**Comment by dorkmo** [comment]
definitely room to speculate one way or the other about network growth.
In theory, adding plots will become more expensive than the expected lifetime profits of each new plot. Then growth would slow.
Once growth slows, energy usage would be expected to fall back toward node and farmer use.
This will also be impacted by the transaction fee market going into the future.
I think it'd be nice to do the math on the energy/environmental impact per TB of hard drive creation in general.
